docs/quickstart_guide_david.md

```diff
@@ -0,0 +1,136 @@
+[Back to Landing Page](overview_david.md)
```

Comment:
```suggestion
```
This isn't necessary. The documentation theme provides this link automatically.

Reply:
Thank you for pointing this out!